applications/emocam/emocam1.js

```diff
@@ -84,11 +86,19 @@ var webWindow;
     var propList = ["headRotation", "headType"];
     handlerId = MyAvatar.addAnimationStateHandler(function (props) {
         if (Date.now() - lastDataArrived < 2000) {
+            if (isProceduralblinkingEnabled) {
+                MyAvatar.hasProceduralBlinkFaceMovement = false;
+                isProceduralblinkingEnabled = false;
+            }
             return {
                 headRotation: headTransform,
                 headType: 4
             };
         } else {
+            if (!isProceduralblinkingEnabled) {
+                MyAvatar.hasProceduralBlinkFaceMovement = true;
+                isProceduralblinkingEnabled = true;
```

Comment:
Could this re-enable procedural blinking when it's not desired?

Would it make sense to have a way to suppress procedural eye blinking without changing the setting?

Reply:
Thanks! I will store previous value and then restore it on lack of data and on app shutdown.

Reply:
I added a variable to store/restore the previous state